src/Analyzers/Core/CodeFixes/UseCollectionInitializer/AbstractUseCollectionInitializerCodeFixProvider.cs

```diff
@@ -97,15 +113,16 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
 #endif
                 CodeActionOptions.DefaultCollectionExpressionWrappingLength;
 
+            using var analyzer = GetAnalyzer();
+
             while (originalObjectCreationNodes.Count > 0)
             {
                 var (originalObjectCreation, useCollectionExpression) = originalObjectCreationNodes.Pop();
                 var objectCreation = currentRoot.GetCurrentNodes(originalObjectCreation).Single();
 
-                var matches = UseCollectionInitializerAnalyzer<TExpressionSyntax, TStatementSyntax, TObjectCreationExpressionSyntax, TMemberAccessExpressionSyntax, TInvocationExpressionSyntax, TExpressionStatementSyntax, TForeachStatementSyntax, TVariableDeclaratorSyntax>.Analyze(
-                    semanticModel, syntaxFacts, objectCreation, useCollectionExpression, cancellationToken);
```

Comment:
no need for consumption sides of the helper type ot have to have so much noise.  this was getting particularly awful as i needded to add 'IfStatementSyntax' here (and all the other places).